# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gamescope/hotfix-use-system.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gamescope.git/meson.build.vanilla	2026-09-03 00:24:47.375243176 +0300
+++ gamescope.git/meson.build	2026-09-03 00:25:08.000171977 +0300
@@ -6,15 +6,9 @@
   default_options: [
     'cpp_std=c++20',
     'warning_level=2',
-    'force_fallback_for=libliftoff,vkroots',
   ],
 )
 
-fallbacks = get_option('force_fallback_for')
-if not (fallbacks.contains('libliftoff') and fallbacks.contains('vkroots'))
-  error('!!!"force_fallback_for" is missing entries!!!\n\tPlease do not remove entries from force_fallback_for if you are packaging the project.\n\tWe pull in these projects at specific commits/forks/builds for a reason.\n\tIf you are not packaging, remove this line to continue.')
-endif
-
 add_project_arguments([
   '-DWLR_USE_UNSTABLE',
 ], language: 'cpp')

